File: Code Academy 2017/2. Bases de données/3. MongoDB: Utiliser les drivers natifs/connect.py
#!/usr/bin/env python

from pymongo import MongoClient

client = MongoClient()
client = MongoClient("127.0.0.1", 27017)
db = client.LXF

sampleData = db.sampleData

item = sampleData.find_one()

print(item['_id'])
print(item['x'])

File: Code Academy 2017/2. Bases de données/3. MongoDB: Utiliser les drivers natifs/connect.rb
#!/usr/bin/env ruby

require "rubygems"
require "mongo"
include Mongo

$client = Mongo::Client.new(['127.0.0.1:27017'], :database => 'LXF')
Mongo::Logger.logger.level = ::Logger::ERROR
$collection = $client[:someData]
puts 'Connected with version:' 
puts Mongo::VERSION